test/server/chat-plugins/help.js

```diff
@@ -15,10 +15,17 @@ const Help = new Responder({
 
 describe('Help', function () {
 	it('should only return true on added regexes', function () {
+		const roomFaqs = require('../../../.server-dist/chat-plugins/room-faqs').roomFaqs;
+		const room = Help.getRoom();
+		if (room) { // if there's no room, this will fail, but if there's no room this test doesn't matter anyway
```

Comment:
unit tests don't use the normal room data (or config data), they have their own clean copies of lobby and staff. this room will never exist.